up-l3/udiscovery/v3/README.adoc

```diff
@@ -206,7 +204,7 @@ NOTE: It is *STRONGLY RECOMMENDED* to scope your property names to avoid namespa
 
 === Database Access
```

Comment:
This should not be in the specification as it is an implementation detail that is not part of the API specifications (more on implementation detail for a given DS. Please remove, same with policy. I don't think we should put database specific details here. It could be a best practice section somewhere else.